killrvideo/comments/comments_events_kafka.py

```diff
@@ -8,7 +9,7 @@ class CommentsPublisher(object):
     """Provides methods that publish events associated with the Comments Service."""
 
     def __init__(self):
-        self.producer = KafkaProducer(bootstrap_servers='10.0.75.1:9092',
+        self.producer = KafkaProducer(bootstrap_servers=os.getenv('KILLRVIDEO_KAFKA', 'kafka'),
```

Comment:
Maybe, KILLRVIDEO_KAFKA_HOST or anything like that? KILLRVIDEO_KAFKA doesn't sound obvious enough.